test/functional/ui/screen.lua

```diff
@@ -212,7 +213,7 @@ function Screen:attach(options, session)
   if options.ext_linegrid == nil then
     options.ext_linegrid = true
   end
-
+  self.pwd = vim.fs.normalize(vim.uv.cwd() or '')
```

Comment:
this gets the dir from the test runner which is a bit confusing. and highlights a missing case of the implementation : whenever a UI attaches, it should receive the current dir of the nvim process immediately.

Reply:
Done e9e1e15812d56273df4e261437985d186f603aa7